packages/eas-cli/src/metadata/errors.ts

```diff
@@ -55,7 +55,14 @@ export function logMetadataValidationError(error: MetadataValidationError): void
   Log.error(chalk.bold(error.message));
   if (error.errors?.length > 0) {
     // TODO(cedric): group errors by property to make multiple errors for same property more readable
-    Log.log(error.errors.map(err => `  - ${chalk.bold(err.dataPath)} ${err.message}`).join('\n'));
+    Log.log(
+      error.errors
+        .map(err => {
+          const dataPath = err.path.join('.');
+          return `  - ${chalk.bold(`$.${dataPath}`)} ${err.message}`;
+        })
+        .join('\n')
+    );
   }
```

Comment:
Refactor to avoid using map inside Log.log:
```
for (const err of error.errors) {
  Log.log(...);
}
```

Reply:
The main reason I used 1 log is performance, but I might be wrong here. I'll change it to a simple loop.

Reply:
Performance shouldn't be a problem here :)